src/app/test-runners/playwright/record-your-first-replay/page.md

```diff
@@ -84,29 +80,26 @@ set REPLAY_API_KEY=<your_api_key>
 ## Update your configuration
 
 ```js {% fileName="playwright.config.ts" highlight=["2-5","10-13","17-20"] lineNumbers=true %}
-import { PlaywrightTestConfig, devices } from '@playwright/test'
-import {
-  createReplayReporterConfig,
-  devices as replayDevices,
-} from "@replayio/playwright";
-import 'dotenv/config'
+import { PlaywrightTestConfig, devices } from "@playwright/test";
+import { devices as replayDevices, replayReporter } from "@replayio/playwright";
+import "dotenv/config";
 
 const config: PlaywrightTestConfig = {
   reporter: [
-    createReplayReporterConfig({
+    replayReporter({
       apiKey: process.env.REPLAY_API_KEY,
       upload: true,
     }),
-    ['line'],
+    ["line"],
   ],
   projects: [
     {
-      name: 'replay-chromium',
-      use: { ...replayDevices['Replay Chromium'] },
+      name: "replay-chromium",
```

Comment:
This naming pattern may cause screenshot snapshot tests to fail. We should talk about this.